cni/network/multitenancy_test.go

```diff
@@ -66,6 +70,18 @@ func (c *MockCNSClient) GetNetworkContainer(ctx context.Context, orchestratorCon
 }
 
 func (c *MockCNSClient) GetAllNetworkContainers(ctx context.Context, orchestratorContext []byte) ([]cns.GetNetworkContainerResponse, error) {
+	// check if getAllNetworkContainersConfiguration orchestratorContext is nil using GetAllNetworkContainers()
+	if c.getAllNetworkContainersConfiguration.orchestratorContext == nil {
+		// if orchestratorContext is same, attach CNSClientError code and error with Unsupported API and try old CNS API GetNetworkContainer() later
```

Comment:
if c.getAllNetworkContainersConfiguration.orchestratorContext is nil why is it being compared to orchestratorContext below?